src/components/LoginForm/index.test.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import { render, fireEvent, waitFor, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import 'mutationobserver-shim';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import LoginForm from './index';
+
+const statusErrorUnprocessableEntity = 422;
+
+const mockApiResponseError = {
+  status: 'error',
+  errors: ['Invalid login credentials']
+};
+
+const server = setupServer(
+  rest.post(`${process.env.REACT_APP_BASE_API_URL}/users/sign_in`, (req, res, ctx) =>
+    res(ctx.status(statusErrorUnprocessableEntity), ctx.json(mockApiResponseError))
+  )
+);
+
+beforeAll(() => server.listen());
+afterEach(() => server.resetHandlers());
+
+beforeEach(() => {
+  jest.resetModules();
+});
+
+afterAll(() => {
+  server.close();
+});
+
+describe('LoginForm', () => {
+  beforeEach(() => {
+    render(
+      <Router>
+        <LoginForm />
+      </Router>
+    );
+  });
+
+  test('Invalid credentals', async () => {
+    const inputEmail = screen.getByTestId('email');
```

Comment:
Avoid using testId. You should use other selectors when possible:
https://testing-library.com/docs/guide-which-query/

For inputs I'd use `getByLabelText`
For buttons you can use `getByRole` or `getByTitle` (I think it works, not sure though)

Reply:
I have used the selectors that you mention in previous commits but they made me change them to these